### ISSUE TITLE
Deprecated conversion float

### ISSUE THREAD

**Issue by zacharytyhacz** [MacroMan/rate-limiter#1]
Hello MacroMan, thank you for this simple rate limiting class, it's awesome.

Upon running the `example.php`, I get this warning:

```log
PHP Deprecated:  Implicit conversion from float 999999.9999821186 to int loses precision in /Users/zact/Projects/my-project/vendor/macroman/rate-limiter/src/RateLimiter/Limiter.php on line 55
```

It points to this: https://github.com/MacroMan/rate-limiter/blob/master/src/RateLimiter/Limiter.php#L55

my php version:
```
PHP 8.1.2 (cli) (built: Jan 21 2022 04:34:05) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.2, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.2, Copyright (c), by Zend Technologies
```

my composer.json:

```json
{
    "name": "zac/test-website",
    "type": "project",
    "require": {
        "vlucas/phpdotenv": "^5.4",
        "mailgun/mailgun-php": "^3.5",
        "symfony/http-client": "^6.0",
        "nyholm/psr7": "^1.4",
        "macroman/rate-limiter": "^1.0"
    },
    "authors": [
        {
            "name": "omitted",
            "email": "omitted"
        }
    ]
}
```

**Comment by MacroMan** [comment]
Hi, thanks for reporting. I'll take a look tomorrow morning UK time when I'm back at my desk.

**Comment by MacroMan** [comment]
Fixed and released as version 1.0.3. Please update your composer package.
